.eslintplugin/index.js

```diff
@@ -0,0 +1,4 @@
+exports.rules = {
+  'ban-types-eventually': require('@typescript-eslint/eslint-plugin/dist/rules/ban-types')
```

Comment:
do we have to reach into `dist`? Why not

```suggestion
  'ban-types-eventually': require('@typescript-eslint/eslint-plugin').rules['ban-types'],
```

?

Reply:
That should work - reaching directly for the rule means only the actual files that are required are `require`d. I don't expect it to cripple anything, but by importing the `index.js` we do technically parse all the other rules (and run any top-level code that might be around).

Reply:
we already use `@typescript-eslint/eslint-plugin`, so one or the other will be a cache hit